src/main/twirl/views/sectionForm.scala.html

```diff
@@ -33,7 +33,7 @@ <h1 class="heading-medium error-summary-heading" id="error-summary-heading">
                     </p>
                 </div>
                 }
-                @app.formSection.fields.map(_.renderFormInput(app, answers, errs, hints))
+                @app.formSection.fields.map(_.renderFormInput(app.formSection.questionMap, answers, errs, hints))
```

Comment:
Not sure if it is here but something odd has happened to the orange outline associated with the mark as complete checkbox.  Probably doesn't need to hold up the technical guts of this pr but is probably a high priority demo fix...
![image](https://cloud.githubusercontent.com/assets/16717302/20675444/2e08c532-b584-11e6-82d4-55b6c7e44d8d.png)